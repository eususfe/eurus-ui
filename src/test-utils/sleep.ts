/*
 * @Author: jiangxd
 * @Date: 2022-09-23 14:06:55
 * @LastEditTime: 2022-09-23 14:07:04
 * @LastEditors: jiangxd
 * @Description:
 * @FilePath: /eurus-ui/src/test-utils/sleep.ts
 */
const sleep = (time = 0) => new Promise(resolve => setTimeout(resolve, time));

export default sleep;
