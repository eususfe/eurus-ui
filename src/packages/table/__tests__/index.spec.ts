import { mount } from '@vue/test-utils';
import { ETable } from '..';

describe('table', () => {
  it('table snapshot', () => {
    const wrapper = mount(ETable);
    expect(wrapper.html()).toMatchSnapshot();
    wrapper.unmount();
  });
});
