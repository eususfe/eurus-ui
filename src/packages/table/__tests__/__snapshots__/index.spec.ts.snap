// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`table > table snapshot 1`] = `
"<div class=\\"eu-table eu-table-size-large eu-table-border eu-table-hover eu-table-empty\\">
  <!---->
  <div class=\\"eu-spin\\">
    <!---->
    <div class=\\"eu-table-container\\">
      <div class=\\"eu-scrollbar eu-scrollbar-type-embed\\" style=\\"height: 100%;\\">
        <div class=\\"eu-scrollbar-container eu-table-content eu-table-content-scroll-x\\">
          <table class=\\"eu-table-element\\" cellpadding=\\"0\\" cellspacing=\\"0\\">
            <colgroup></colgroup>
            <thead></thead>
            <tbody>
              <tr class=\\"eu-table-tr eu-table-tr-empty\\">
                <td class=\\"eu-table-td\\"><span class=\\"eu-table-cell eu-table-cell-align-left\\"><!----><!----><span class=\\"eu-table-td-content\\"><div class=\\"eu-empty\\"><div class=\\"eu-empty-image\\"><span class=\\"eu-icon\\" role=\\"link\\" tabindex=\\"0\\" style=\\"font-size: 20px;\\"><svg width=\\"48\\" height=\\"48\\" viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" stroke-width=\\"4\\"><path d=\\"M24 5V11M31 12L35 8M17 12L13 8M41.5 30L28 30C28 30 27 33 24 33C21 33 20 30 20 30H6.5M40 41H8C6.89543 41 6 40.1046 6 39L6 30.5407C6 30.1866 6.09402 29.8388 6.27244 29.533L12.4212 18.9923C12.7796 18.3778 13.4374 18 14.1487 18H33.8513C34.5626 18 35.2204 18.3778 35.5788 18.9923L41.7276 29.533C41.906 29.8388 42 30.1866 42 30.5407V39C42 40.1046 41.1046 41 40 41Z\\" stroke-linecap=\\"butt\\"></path></svg></span>
        </div>
        <div class=\\"eu-empty-description\\">暂无数据</div>
      </div></span></span></td>
      </tr>
      </tbody>
      <!---->
      </table>
    </div>
    <!--v-if-->
    <!--v-if-->
  </div>
</div>
<!---->
<!---->
<!---->
</div>
</div>"
`;
